### PR TITLE
Added jv-lottery solution

### PR DESCRIPTION
**Pull request by Stanislav-Viner**

### CODE REVIEW

File: src/main/java/core/basesyntax/Lottery.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String str = new ColorSupplier().getRandomColor();
+        int index = new Random().nextInt(100) + 1;
```
Comment (okuzan) [inline src/main/java/core/basesyntax/Lottery.java:8]: make ColorSupplier (and Random) class field, not a local variable
make 100 a class constant
rename uninformative `str` to `color`

**Review by okuzan — CHANGES_REQUESTED** [review]
Good job on your first attempt!

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 2]
```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```
Comment (olekskov) [inline src/main/java/core/basesyntax/ColorSupplier.java:7]: same here

File: src/main/java/core/basesyntax/Application.java [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(new Lottery().getRandomBall());
+        }
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Application.java:7]:
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 4]
```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/ColorSupplier.java:5]: You misunderstood the previous comment a bit. No need to move the index calculation here. Just the Random object itself.

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 5]
```diff
@@ -1,7 +1,11 @@
...
+    private int index = new Random().nextInt(Color.values().length);
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/ColorSupplier.java:6]:
```suggestion
    Random random = new Random();
```

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 6]
```diff
@@ -1,7 +1,11 @@
...
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[index].name();
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/ColorSupplier.java:9]:
```suggestion
        return Color.values()[random.nextInt(Color.values().length)].name();
```

File: src/main/java/core/basesyntax/Lottery.java [hunk 7]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private int number = new Random().nextInt(MAX_NUMBER) + 1;
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Lottery.java:8]: same as previous class

**Review by kozhukhovsky — CHANGES_REQUESTED** [review]

**Review by kozhukhovsky — APPROVED** [review]
Good job!

**Pull request closed without merging**